### DISCUSSION TITLE
Como conseguir o primeiro trampo

### DISCUSSION THREAD

**Discussion by kaio0w** [backend-br/forum#142] [Q&A]
Olá comunidade,

Acabei de concluir minha graduação e ganhei experiência de um estágio. No entanto, agora estou enfrentando o desafio de entrar no mercado de trabalho de forma mais sólida, atrás do primeiro trampo né. Gostaria de ouvir sugestões e conselhos daqueles que já passaram por esse processo.
Venho dando matchs de 100% com algumas vagas mas na hora de ser chamado para uma entrevista nada

**Comment by gustavofreze** [comment]
É importante ressaltar que não há uma fórmula exata para garantir o primeiro emprego como engenheiro de software. No entanto, as estratégias mencionadas aqui podem oferecer orientações valiosas e potencialmente ajudar a melhorar suas chances nesse processo.

- **Currículo**: Personalize seu currículo para realçar suas habilidades técnicas e experiência de estágio. Destaque projetos relevantes e conquistas específicas para destacar seu potencial como engenheiro de software.

- **Networking**: Conecte-se com profissionais da área, participe de eventos e use plataformas online para expandir sua rede. O networking pode abrir portas e proporcionar oportunidades de entrevistas.

- **Preparação para entrevistas**: Pratique respostas para perguntas comuns de entrevistas técnicas e comportamentais. Demonstre sua paixão pela área e sua vontade de aprender.

- **Portfólio**: Desenvolva projetos independentes ou contribua para projetos de código aberto. Ter um portfólio sólido pode chamar a atenção dos recrutadores.

- **Solicite feedback**: Se possível, peça feedback às empresas que não o escolheram após as entrevistas. Isso pode fornecer insights valiosos para melhorar suas habilidades e abordagem.

**Comment by igorpereirag** [comment]
Foque nessas três coisas:

- Projetos próprios para ter experiência  e demonstrar isso nas entrevistas 

- networking

- Um LinkedIn/ currículo bem feito e configurado  com as palavras chaves da vaga que procura

**Comment by SavioHaistem** [comment]
Além das dicas a cima, tenho achado recentemente alguns projetos open source aqui no github que devs compartilham métodos para tal façanha

este é um que gostei bastante, tem todo um forúm de ajuda e subprojetos para ajudar a estudar o que for preciso:

https://www.techinterviewhandbook.org/software-engineering-interview-guide/

**Comment by hebcodee** [comment]
Estuda bastante e faz network cara, com o tempo oportunidades surgem e voce tem que agarralas tendo conhecimento tecnico
